ipython_notebook_config.py

```diff
@@ -7,31 +9,25 @@
 c.NotebookApp.profile = u'default'
 c.IPKernelApp.matplotlib = 'inline'
 
-import os
-import yaml
-
 headers = {
     'X-Frame-Options': 'ALLOWALL',
 }
 c.NotebookApp.allow_origin = '*'
 c.NotebookApp.allow_credentials = True
 
-config_file_path = '/import/conf.yaml'
 # In case this Notebook was launched from Galaxy a config file exists in /import/
 # For standalone usage we fall back to a port-less URL
-if os.path.exists( config_file_path ):
-    with open( config_file_path ,'r') as handle:
-        conf = yaml.load(handle)
-    c.NotebookApp.base_url = '/ipython/%d/' % conf['docker_port']
-    c.NotebookApp.webapp_settings = {'static_url_prefix':'/ipython/%d/static/' % conf['docker_port']}
+if os.environ.get('DOCKER_PORT', 'none') == 'none':
+    c.NotebookApp.base_url = '/ipython/'
+    c.NotebookApp.webapp_settings = {'static_url_prefix': '/ipython/static/'}
+else:
+    c.NotebookApp.base_url = '/ipython/%d/' % os.environ['DOCKER_PORT']
+    c.NotebookApp.webapp_settings = {'static_url_prefix': '/ipython/%d/static/' % os.environ['DOCKER_PORT']}
 
-    if 'notebook_password' in conf:
-        c.NotebookApp.password = conf['notebook_password']
+if os.environ.get('NOTEBOOK_PASSWORD', 'none') != 'none':
```

Comment:
Explicit is better than implicit! 
I would like to have it like *galaxy-stable*. 0.[latest] should be master and we should have a `:dev` branch. Thoughts?